typings/internalBinding/fs.d.ts

```diff
@@ -94,10 +94,11 @@ declare namespace InternalFSBinding {
   function fstat(fd: number, useBigint: boolean, usePromises: typeof kUsePromises): Promise<Float64Array | BigUint64Array>;
   function fstat(fd: number, useBigint: true, usePromises: typeof kUsePromises): Promise<BigUint64Array>;
   function fstat(fd: number, useBigint: false, usePromises: typeof kUsePromises): Promise<Float64Array>;
+  function fsync(fd: number): void;
```

Comment:
We still have the callback and promise versions. Please keep the existing ones and just add a new one (of this line)

Reply:
Sorry, it was modified incorrectly. Thank you. 😢